src/components/button/button.njk

```diff
@@ -0,0 +1,7 @@
+{% macro button(slug, title, type='primary') %}
+  {% if slug %}
+    <a href="{{ slug }}" class="button button--{{ type }}">
```

Comment:
But it's not a button, it's an anchor tag?

Reply:
I realise now that the `if` statement is probably there for a reason, so it would render a `<button>` in the `else`, but it's not implemented yet?